### DISCUSSION TITLE
Import from OneNote

### DISCUSSION THREAD

**Discussion by ghnp5** [outline/outline#7126] [Ideas]
Hello,

I'm considering using Outline (self-hosted), to go away from using OneNote.

Is there a way to import all sections and pages from OneNote notebooks, into Outline?

Thank you very much!

**Comment by tommoor** [comment]
Nothing is built that I'm aware of, but you can use the [API](https://www.getoutline.com/developers) – in particular the `documents.import` endpoint to automate something like this.
